Makefile

```diff
@@ -130,6 +130,12 @@ ib-init-package:
 		--set REGISTRY_IMAGE="ironbank/opensource/docker/registry-v2" \
 		--set REGISTRY_IMAGE_TAG="2.8.2"
 
+# INTERNAL: used to publish the init package
+publish-init-package:
+	@test -s $(ZARF_BIN) || $(MAKE) build-cli
```

Comment:
```suggestion
```

I think I would rather this line be left out similar to how `release-init-package` is.  That way there is less chance a dev build of zarf does something weird.